client/src/components/ContactCitySection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class='header-section'>
+      <div class='h2-header-large'>
+        {{ MESSAGES.CONTACT_YOUR_CITY_HEADER }}
+      </div>
+      <div>
+        {{ MESSAGES.CONTACT_YOUR_CITY_SUBHEADER }}
+      </div>
+    </div>
+    <div class='city-information'>
+      <div class='h2-header'> {{ this.city }}</div>
+      <a class='h2-header' :href='website' target='_blank'>{{ website }}</a>
+      <div class='h2-header'> Tel: {{ phoneNumber }}</div>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+
+import { City } from '../model/states/model/geo_data';
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+
+interface CityInformation {
```

Comment:
nit: I'd rather name this something more specific to its purpose, such as `ContactInfo`.

Reply:
Change to CityContactInfo